grommet/grommet-docs
# Add Google Analytics Tracking / Page Views

#### What does this PR do?
Adds Google Analytics tracking and page views

#### Where should the reviewer start?
`src/index.js` is where the tracker is initialized, but also checkout the utils that I added.  I wasn't sure of the best place to put these, so I made a utils folder within the `src` directory.  If you can think of a better place, please let me know and I'd be happy to change it.

#### What testing has been done on this PR?
I tested it manually by running the docs site and verifying that Google Analytics was capturing page views.

#### How should this be manually tested?
Launch the site locally and open the real-time overview on Google Analytics to see yourself get logged in real time.

#### Any background context you want to provide?
I was asked by Christina and Tracy to help get Google Analytics up and running for Grommet docs.

#### What are the relevant issues?
#274 

#### Screenshots (if appropriate)
N/A
